cmd/inventory/utils_openstack.go

```diff
@@ -201,232 +197,94 @@ func newOpenStackProviderClient(
 	return gophercloudconfig.NewProviderClient(ctx, authOpts)
 }
 
-// configureOpenStackComputeClientsets configures the OpenStack Compute API clientsets.
-func configureOpenStackComputeClientsets(ctx context.Context, conf *config.Config) error {
-	for _, clientConfig := range conf.OpenStack.Services.Compute {
-		creds := clientConfig.UseCredentials
-		namedCreds := conf.OpenStack.Credentials[creds]
+func configureClientset(
```

Comment:
Please add a prefix/suffix like the other functions for the different providers. `configureClientset` is too generic, so it's better to rename it to something Openstack-specific.